src/main/java/org/datadog/jmxfetch/reporter/Reporter.java

```diff
@@ -60,8 +60,6 @@ public void sendMetrics(LinkedList<HashMap<String, Object>> metrics, String inst
             // We need to edit metrics for legacy reasons (rename metrics, etc)
             HashMap<String, Object> metric = new HashMap<String, Object>(m);
 
-            postProcess(metric);
```

Comment:
You can also remove this method's definition, along with `postProcessCassandra` that is not used anymore.